charts/pathfinder-rpc-server/templates/blockchain-indexer-statefulset.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
@JacqueGM Would you have any idea, why this was turned into a StatefulSet and not a Deployment? I cannot see any volume mounts here, why the latter seems to be the preferred solution.